wrappers/src/shared_realm_cs.cpp

```diff
@@ -754,6 +754,20 @@ REALM_EXPORT bool shared_realm_remove_type(const SharedRealm& realm, uint16_t* t
     });
 }
 
+REALM_EXPORT bool shared_realm_remove_all(const SharedRealm& realm, NativeException::Marshallable& ex)
+{
+    return handle_errors(ex, [&]() {
+        realm->verify_in_write();
+
+        auto full_schema = ObjectStore::schema_from_group(realm->read_group());
```

Comment:
This is related to [get_full_schema from core](https://github.com/realm/realm-core/blob/b86d2107fec4246b55af24a3b11e00275575f808/src/realm/object-store/shared_realm.cpp#L329) which is unfortunately a private field right now... I am not sure how many of those checks the function does is crucial beyond caching for sake of performance... Immutability check seems relevant here, but I am unsure; current implementation like this passes both basic and partial schema tests that used to not pass. @nirinchev